tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -366,6 +366,21 @@ public function getUserIdByUserName(string $userName): string {
 		throw new Exception(__METHOD__ . " user with name $userName not found");
 	}
 
+	/**
+	 * using method from core to set share data
+	 * 
+	 * @return void
+	 */
+	public function setLastShareData(): void
+	{
```

Comment:
```suggestion
	public function setLastShareData(): void {
```
I'm surprised codestyle does not pick that up